packages/core-data/src/hooks/use-entity-record.ts

```diff
@@ -0,0 +1,109 @@
+/**
+ * WordPress dependencies
+ */
+import { useQuerySelect } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import { store as coreStore } from '../';
+import { Status } from './constants';
+
+interface EntityRecordResolution {
+	/** The requested entity record */
+	record: Object;
+
+	/** The requested entity record with any edits applied*/
+	editedRecord: Object;
```

Comment:
What is the practical usage of `record` vs `editedRecord`? Why the hook’s consumer would need both in this high-level API?

Reply:
Oh yes, that makes sense – most of the time `editedRecord` is only needed when there are mutations on the table and I also need access to `editEntityRecord` and perhaps `saveEditedEntityRecord`. I think we can remove it from here just fine 👍